CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Review/ReviewBottomNavigationBar.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  ReviewBottomNavigationBar.swift
+//  
+//
+//  Created by David Vizaknai on 24.10.2022.
+//
+
+import UIKit
+
+protocol ReviewBottomNavigationBarDelegate: AnyObject {
+    func didTapMainButton(on navigationBar: ReviewBottomNavigationBar)
+    func didTapSecondaryButton(on navigationBar: ReviewBottomNavigationBar)
+}
+
+final class ReviewBottomNavigationBar: UIView {
+    private let configuration = GiniConfiguration.shared
+    lazy var mainButton: MultilineTitleButton = {
+        let button = MultilineTitleButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.titleLabel?.font = configuration.textStyleFonts[.bodyBold]
+        button.titleLabel?.adjustsFontForContentSizeCategory = true
+        button.layer.cornerRadius = 16
+        button.backgroundColor = UIColor.GiniCapture.accent1
+        button.setTitle(NSLocalizedStringPreferredFormat("ginicapture.multipagereview.mainButtonTitle",
+                                                         comment: "Process button title"), for: .normal)
+        button.addTarget(self, action: #selector(mainButtonClicked), for: .touchUpInside)
+        return button
+    }()
+
+    lazy var secondaryButton: BottomLabelButton = {
+        let addPagesButton = BottomLabelButton()
+        addPagesButton.translatesAutoresizingMaskIntoConstraints = false
+        addPagesButton.configureButton(image: UIImageNamedPreferred(named: "plus_icon") ?? UIImage(),
+                                       name:
+                        NSLocalizedStringPreferredFormat("ginicapture.multipagereview.secondaryButtonTitle",
+                                                        comment: "Add pages button title"))
+        addPagesButton.isHidden = !configuration.multipageEnabled
+        if #available(iOS 13.0, *) {
+            addPagesButton.iconView.tintColor = .GiniCapture.light1
+            addPagesButton.iconView.image = addPagesButton.iconView.image?.withTintColor(.GiniCapture.light1,
+                                                                                         renderingMode: .alwaysTemplate)
+        } else {
+            // Fallback on earlier versions
```

Comment:
[Clarification] Could you please check, I think we have an extension for the earlier versions.

Reply:
Didn't know about the extension. As this is temporary, I didn't want to add an extension just for this. Fixed it